pint/testsuite/pandas_test_interface.py

```diff
@@ -275,8 +304,25 @@ class TestReshaping(base.BaseReshapingTests):
     pass
 
 class TestSetitem(base.BaseSetitemTests):
-    pass
+    @pytest.mark.parametrize('setter', ['loc', None])
+    @pytest.mark.filterwarnings("ignore::pint.UnitStrippedWarning")
```

Comment:
I'd like to keep these warnings popping up unless there's a good reason to silence them?